lessons/Data-Science/histograms-visualize/langs/en-us/index.adoc

```diff
@@ -0,0 +1,334 @@
+
+| Values are clumped around what's typical, but trail off to the right.
+
+| Values are clumped around what's typical, but trail off to the left.
+
+|===
+
+Skew-left distributions look like the toes on your left foot, and skew-right distributions look like the toes on your right foot!
+
+@vocab{Shape} is useful because it enables us to quickly identify and describe trends in data. When we recognize a particular shape, we can immediately draw conclusions about a dataset! We'll dig deeper into that idea in our lesson on @lesson-link{histograms-interpret}.)
+
+*Shape is one of a few key features that sets histograms apart from bar charts!*
+
+@lesson-instruction{
+Complete @printable-exercise{bar-chart-v-histogram.adoc}.
+}
+
+@teacher{
+@printable-exercise{bar-chart-v-histogram.adoc} surfaces two common student misconceptions about bar graphs that @citation{whittaker-jacobbe-2017, "Whittaker and Jacobbe (2017)"} cite in their research.
+
+@centered-image{images/majors-bar-chart.png, 350}
+
+- First, students commonly fail to understand that skew is associated with *distributions of quantitative variables*. This is why a bar graph with its bars arranged in increasing or decreasing order does not display a skewed distribution.
+- Students also commonly believe that the category with the greatest value represents a majority of the responses. In this dataset, that is not the case: the bar representing the college of science does not represent more than half of the students.
+}
+
+Because a bar chart displays categorical data, we can order the bars anyway that we wish. That is not the case with histograms: since quantitative data must follow a natural order, a histogram's bars cannot be re-ordered.
+
+@QandA{
+@Q{What are some ways that bar charts and histograms are alike?}
+@A{}
+@Q{What are some ways that bar charts and histograms are different?}
+@A{}
+}
+
+=== Synthesize
+
+@QandA{
+
+@Q{Envision a skew-left histogram. Where do you think its outliers are? Explain how you know.}
+
+@Q{Why do histograms have shape but bar charts do not?}
+@Q{Because histograms are quantitative, their bars must appear in numerical order. Bar charts show categorical data, meaning that the bars can be arranged in any way.}
+
+}
+
+@teacher{
+
+Want to check student mastery of the content you've just taught? Administer @assessment{histograms-check1-desmos} to get a snapshot of your students' current level of mastery. Make sure you have created a link or code for your class to the assessment.
+
+Alternatively, we offer a compilation of both Checkpoints in @assessment{histograms-cumulative-desmos}.
+}
+
+
+== Choosing the Right Bin Size
+
+@objective{choosing-bin-size}
+
+=== Overview
+Students learn to make histograms from the animals-dataset in @proglang and explore the importance of choosing the right bin size in order for a histogram to show us the shape of the data.
+
+=== Launch
+
+@QandA{
+@Q{What is a bin on a histogram? And what do we already know about them?}
```

Comment:
@retabak please confirm that my wording for this QandA works. Thanks!

Reply:
I like this.